js/grid.custom.js

```diff
@@ -397,8 +397,8 @@ $.jgrid.extend({
                                 
 				str += "</div>";
 				body.append(str);
-				var sopt_menu = $("#sopt_menu");
-                                sopt_menu.addClass("ui-menu ui-widget");
+				sopt_menu = $("#sopt_menu");
+                sopt_menu.addClass("ui-menu ui-widget");
```

Comment:
Yes, I've seen that in a couple of diffs already. I think the mess originally comes from Tony's repo. My whitespace commit in the beginning had fixed that, but we reverted it (which was good in retrospect, because it introduced some other indentation problems..).

Did we decide on tabs vs. spaces already? If so, we could just replace them once globally and set the appropriate jshint flag afterwards. If not, I vote for 4 spaces.